src/main/java/uk/ac/ebi/ampt2d/metadata/SpringDataRestConfig.java

```diff
@@ -113,4 +115,10 @@ public StudyDeprecationAspect studyDeprecationAspect() {
         return new StudyDeprecationAspect();
     }
 
+    @Bean
+    @ConditionalOnProperty(name = "endpoints.studies.date.restricted", matchIfMissing = true)
```

Comment:
I understand what you are doing there now, but please document or we will forget it ;) Same with other aspects.

Reply:
Added in b8b6677.